### ISSUE TITLE
Setting alpha in TOST

### ISSUE THREAD

**Issue by K-Maehashi** [Lakens/TOSTER#81]
Hi authors, thanks for the great package!

I have a quick question--why are we using the [alpha*2](https://github.com/Lakens/TOSTER/blob/master/R/t_TOST.R#L304) for the lower/upper t-tests in t_TOST.R?

Reading the recommended paper ([Equivalence Tests: A Practical Primer for t Tests, Correlations, and Meta-Analyses](https://journals.sagepub.com/doi/epdf/10.1177/1948550617697177)), I wonder we use alpha (p.357, below Equation (4)).

**Comment by arcaldwell49** [comment]
I hate to be blunt, but I would suggest reading the papers by Daniel. Remember, these are *two one-tailed tests*, and with a 5% alpha level we would report a 90% confidence interval.

**Comment by K-Maehashi** [comment]
Thank you for curving out time form your busy schedule. Schuirmann (1987) says "The two one-sided tests procedure turns out to be operationally identical to the procedure of declaring equivalence only if the ordinary 1-2\alpha (not 1-\alpha) confidence interval for \mu_T-\mu_R is completely contained in the equivalence interval."

I wonder why three t-tests in [the function](https://github.com/Lakens/TOSTER/blob/master/R/t_TOST.R) use the same level of alpha. Shouldn't each t-test (lower, upper) have the desired overall significance level of \alpha?
```
  tresult = t.test(x = x,
                   y = y,
                   paired = paired,
                   var.equal = var.equal,
                   mu = mu,
                   conf.level = 1 - alpha*2,
                   alternative = "two.sided")

  low_ttest <- t.test(
    y = y,
    x = x,
    paired = paired,
    var.equal = var.equal,
    alternative = alt_low,
    mu = low_eqbound,
    conf.level = 1-alpha*2
  )

  high_ttest <- t.test(
    y = y,
    x = x,
    paired = paired,
    var.equal = var.equal,
    alternative = alt_high,
    mu = high_eqbound,
    conf.level = 1-alpha*2
  )
```

I think the output is correct only because the package uses the p-values from low_ttest and high_ttest, and create the CI from tresults.
```
lower.ci = c(tresult$conf.int[1], cohen_res$dlow),
upper.ci = c(tresult$conf.int[2], cohen_res$dhigh)
```

**Comment by arcaldwell49** [comment]
Again, I would strongly suggest reading the papers, and possibly the documentation for `t.test`, in depth because I don't think you have a strong grasp of what the functions are actually doing.

Also, as you can see in the code, the `alpha` argument within `t.test` only affects the confidence level which is why that argument is modified in the first place. You'll see the alpha level remains unchanged when generating the text interpretation of the p-values. The `tresult` object is used to extract the CI since the lower/upper bound would be missing from a `t.test` result when the `alternative` is not set to "two.sided".

TL;DR, you're worried about an argument to function within a function that has no bearing on the provided interpretation from said function.

**Comment by arcaldwell49** [comment]
To break it down further, an example. Let us say we want to perform an equivalence test using TOST on the `sleep` data. If I am not interested in the exact p-value from TOST, I can just print the confidence interval from the two tailed test

As you can see from the output below, we can judge whether at a 5% alpha level if the estimated mean difference is within our equivalence bound. This is what tresult is doing within the function you quoted; the other `conf.int` settings are irrelevant.

```
> t.test(extra ~ group, data = sleep, paired = TRUE, 
+        conf.level = .9)

	Paired t-test

data:  extra by group
t = -4.0621, df = 9, p-value = 0.002833
alternative hypothesis: true mean difference is not equal to 0
90 percent confidence interval:
 -2.2930053 -0.8669947
sample estimates:
mean difference 
          -1.58 
``` 

Now, we may want an exact p-value for the equivalence test (let's set it to 2.5). That is what the other two `t.test` results in the code are for!

```
> low = t.test(
+   extra ~ group,
+   data = sleep,
+   paired = TRUE,
+   alternative = "greater", # direction of test
+   mu = -2.5, # lower bound
+   conf.level = .9 # doesn't matter
+ )
> upper = t.test(
+   extra ~ group,
+   data = sleep,
+   paired = TRUE,
+   alternative = "less",# direction of test
+   mu = 2.5, # upper bound
+   conf.level = .9 # doesn't matter
+ )
> 
> max(low$p.value, upper$p.value)
[1] 0.02111686
```

This is how any of the equivalence test functions in TOSTER work. For example:

```
> test = TOSTER::simple_htest(  extra ~ group,
+                data = sleep,
+                paired = TRUE,
+                alternative = "eq",
+                mu = 2.5)
> test$p.value
[1] 0.02111686
> test$conf.int
[1] -2.2930053 -0.8669947
attr(,"conf.level")
[1] 0.9
```
